Description: Enable SSL support for tools.
 Add WITH_SSL preprocessor macro it SSL support is enabled. Otherwise tools
 would abort it they see an SSL URL.
Author: Michael Fladischer <FladischerMichael@fladi.at>
Last-Update: 2014-10-17
Bug-Debian: https://bugs.debian.org/764080
Forwarded: https://github.com/alanxz/rabbitmq-c/pull/220

--- a/tools/CMakeLists.txt
+++ b/tools/CMakeLists.txt
@@ -72,6 +72,10 @@
   endif(XMLTO_FOUND)
 endif()
 
+if (ENABLE_SSL_SUPPORT)
+  add_definitions(-DWITH_SSL=1)
+endif()
+
 install(TARGETS amqp-publish amqp-get amqp-consume amqp-declare-queue amqp-delete-queue
 	RUNTIME DESTINATION bin
 	LIBRARY DESTINATION lib
